tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -52,6 +52,10 @@ public class FolderComboBoxItem
 
     private readonly ObservableCollection<ExplorerItem> _dataSource = new();
 
+    private readonly Guid _activityId;
+
+    public Guid ActivityId => _activityId;

```

Comment:
Since `_activityId` is initialized in the constructor, we can reduce this to:

```suggestion
    public Guid ActivityId { get; }
```